src/nickel.rs

```diff
@@ -81,7 +81,7 @@ impl<D: Sync + Send + 'static> Nickel<D> {
     /// as soon as any regular handler returned an error
     ///
     /// A error handler is nearly identical to a regular middleware handler with the only
-    /// difference that it takes an additional error parameter or type `NickelError.
+    /// difference that it takes an additional error parameter of type `NickelError.
     ///
```

Comment:
I think this doccomment has diverged slightly from the original ErrorHandler, this line doesn't really ring true so well anymore. Maybe `See the <link>ErrorHandler</link> trait for more information.` is more appropriate now? And then we can better document the trait there?